script/tool/lib/src/build_examples_command.dart

```diff
@@ -48,12 +48,12 @@ class BuildExamplesCommand extends PackageLoopingCommand {
     ProcessRunner processRunner = const ProcessRunner(),
     Platform platform = const LocalPlatform(),
   }) : super(packagesDir, processRunner: processRunner, platform: platform) {
-    argParser.addFlag(kPlatformLinux);
-    argParser.addFlag(kPlatformMacos);
-    argParser.addFlag(kPlatformWeb);
-    argParser.addFlag(kPlatformWindows);
-    argParser.addFlag(kPlatformWinUwp);
-    argParser.addFlag(kPlatformIos);
+    argParser.addFlag(platformLinux);
+    argParser.addFlag(platformMacOS);
+    argParser.addFlag(platformWeb);
+    argParser.addFlag(platformWindows);
+    argParser.addFlag(platformWinUwp);
```

Comment:
Wouldn't you want `platformWinUWP`?

Reply:
UWP is three letters, not two.

Reply:
"iOS" is 3 letters too.  It's not a 2 letter acronym.  If this is what you all decided upon that's fine but I think we need to do a better job describing the algorithm like the Java style guide does.  I think the way that is written leaves zero ambiguities.

Reply:
UWP is unambiguously not a two-letter acronym (which is what the Dart style guide makes an exception for). `platfromWinUWP` is therefore unambiguously against Dart style.